lib/plugin/allure.js

```diff
@@ -215,7 +215,13 @@ module.exports = (config) => {
     }
 
     err.message = err.message.replace(ansiRegExp(), '');
-    reporter.endCase('failed', err);
+    if (reporter.getCurrentTest()) {
+      reporter.endCase('failed', err);
+    } else if (err) {
```

Comment:
isn't `else` enough? or we have another possibility 😄

Reply:
i suppose it is enough, happy to change it if you think thats best

Reply:
https://github.com/Codeception/CodeceptJS/pull/2248/commits/69c6c960420ec3b32894e1c4bb6afe4cd7b4e23d